Specs/Number/Arabic/NumberModel.json

```diff
@@ -187,47 +187,62 @@
     ]
   },
   {
-    "Input": "مائة وثلاثة وثلثين",
```

Comment:
Why are specific tests being modified? We should try to make sure that there is some level of parity with English (same cases) to guarantee coverage. 
If you need to add additional cases, or Arabic-specific ones, please append them at the end of the appropriate json files.